src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,37 @@
+@import "../utils/variables";
+
+.stars {
+  justify-content: space-between;
+  margin-bottom: 24px ;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.